### DISCUSSION TITLE
Kibana Not Starting

### DISCUSSION THREAD

**Discussion by xaqpuyrthnbg** [Security-Onion-Solutions/securityonion#2027] [Unsupported Versions] [answered]
I upgraded 2.3.10 last night and came back to it this morning.  Kibana normally takes a little time to come up, but it has be more than several hours and a reboot and it is not coming back up.  I get the normal error "Kibana server is not ready yet" but so-status indicates that everything is "ok" across the board.  

Any ideas?  I haven't really ever had an issue with this in the past.  

Thanks in advance.

**Comment by weslambert** [comment]
Could it be because it might be trying to migrate the Kibana index?  Do you notice anything that stands out if you tail `/opt/so/log/kibana/kibana.log`?

**Reply by xaqpuyrthnbg** [reply]
No, it just said it was migrating the index, but it shouldn't take ~10 hours to migrate I wouldn't think?

**Reply by weslambert** [reply]
It shouldn't, so you may need to manually take some steps to rectify.

What is the output of the following?

`curl -s localhost:9200/_cat/indices | grep kibana`
`curl -s localhost:9200/_aliases | jq .`

**Reply by xaqpuyrthnbg** [reply]
curl -s localhost:9200/_cat/indices | grep kibana output: 

green open  .kibana_2                 VJ-3p98YQNO895gClBcZxA 1 0    2296   15 961.1kb 961.1kb
green open  .kibana_1                 ndOumRzeRe-QCsf9oO905g 1 0    2276   12 969.5kb 969.5kb
green open  .kibana_3                 d_rpM5AqQrOHVPDbNatYjw 1 0       0    0    208b    208b

curl -s localhost:9200/_aliases | jq . output:

https://pastebin.com/QJAWjh3a

**Comment by zaphode** [comment, marked as answer]
I ran into this same problem. 
`tail /opt/so/log/kibana/kibana.log`

The log entry suggests deleting '.kibana_3' to fix it but this just gets you to the next error which looks like some sort of Kibana/Elastic version mismatch (maybe trying to load a newer dashboard version into an older version of Kibana). 

Likely not the best fix but this got it working on a few installs that soup broke.
I don't know that I would try this on a production install
--------------------------------------------------------------------------------------
 Step 1) Find Kibana Indices:
`curl -s localhost:9200/_cat/indices | grep kibana`

Step 2) Delete all indices:
```
curl -XDELETE localhost:9200/.kibana_1
curl -XDELETE localhost:9200/.kibana_2
curl -XDELETE localhost:9200/.kibana_3
```

Step 3) Reboot Security Onion:
`reboot`

Step 4) Reload Security Onion Kibana dashboards:
`so-kibana-config-load`

Step 5 ) Life is good :)

**Reply by xaqpuyrthnbg** [reply]
Awesome!  I have to admit that soup broke Kibana and the hive for me and it just seemed jenky overall; so I ended up doing a nuke and pave and its been much better.  Curious to see how the next version upgrade goes.
